src/ContainerFactory.ts

```diff
@@ -1,10 +1,11 @@
 import { Container } from 'inversify';
 import { buildProviderModule } from 'inversify-binding-decorators';
 import { CookieConfig, SessionMiddleware, SessionStore } from 'ch-node-session-handler';
-import IORedis = require('ioredis');
-import { RequestHandler } from 'express';
 import { getEnvOrDefault } from './utils/EnvironmentUtils';
 import { AuthMiddleware } from './middleware/AuthMiddleware';
+import * as IORedis from 'ioredis'
+import * as kafka from 'kafka-node'
+import { EmailService } from './modules/email-publisher/EmailService'
```

Comment:
The intention is to extract `modules/email-publisher` into separate repository as an external NPM module.

Reply:
Is it going to be used by other services other than ours as far as we know?

Reply:
No clear candidate right now but it seems to be good candidate for reuse.